jax/experimental/compilation_cache/file_system_cache.py

```diff
@@ -54,6 +54,7 @@ def _evict_entries_if_necessary(self, key: str, value: bytes) -> bool:
     if new_file_size >= self._max_cache_size_bytes:
       return False
 
+    #TODO: Refactor this section so in the future the whole directory doesn't need to be checked
```

Comment:
TODO formatting: use `# TODO(colemanliyah): blah blah` (space after `#`, put your username in the TODO to indicate you're likely to address it, or can at least answer questions about it)

Also, you can remove the "in the future", all TODOs are in the future :D